lib/handleRequest.js

```diff
@@ -118,6 +119,11 @@ function preHandlerCallback (err, reply) {
         reply.send(payload)
       }
     }).catch((err) => {
+      if (isSystem(err)) {
+        reply.res.statusCode = 500
+        reply.res.end()
```

Comment:
I’m -1 here. If we are doing this, we just throw it with the assumption that the process is going to crash afterwards. If we return a 500, then this is useless.

Reply:
What's your expectation? The server should exit and be no longer functional right? I close the socket connection.